### ISSUE TITLE
Deload Feature

### ISSUE THREAD

**Issue by pepablock** [JonnyBeeGod/nSuns-5-3-1-iOS-issues#88]
Hi, love the app, I was thinking it would be cool if there was an option to deload (on how to deload: [https://www.reddit.com/r/nSuns/comments/ao9t7z/a_post_about_deload_weeks_how_to_do_them_when_to/](https://www.reddit.com/r/nSuns/comments/ao9t7z/a_post_about_deload_weeks_how_to_do_them_when_to/)) an entire week and then comeback to normal. Maybe when you swipe left to delete logged workout you might add swipe right to deload the whole week.

**Comment by JonnyBeeGod** [comment]
Good idea. Since there are many ways how to to deload (look e.g. here https://www.reddit.com/r/nSuns/comments/bznngf/how_to_deload/) what do you think would be best or used most often?  
- Lowering intensity is the easiest by keeping the template as is and just lowering the percentages by a certain amount which could also be user-configured.
- Lowering volume is harder because how would you select which sets to remove? Probably the last sets of each day? 
- ...

**Comment by pepablock** [comment]
As you said the easiest way would be just lowering the % of all lifts.

Drop the intesity of all lifts by 50% as a default value, if a lifter for whatever reason don't want to cut by 50% let them put any other input they want to.

Lowering volume would be indeed a bit trickier. Or you can do a combination of both.

https://renaissanceperiodization.com/expert-advice/dr-mike-israetel-compilation

Here's a good article about deload from Mike. It's basically what you mentioned above

**Comment by JonnyBeeGod** [comment]
Another great feedback from a user: Deloading should not affect the graphs (or it should be marked as deloaded in the graph) otherwise progress chart will look skewed
